test/e2e/generator/random.go

```diff
@@ -57,8 +57,10 @@ func (uc uniformChoice) Choose(r *rand.Rand) interface{} {
 }
 
 // weightedChoice chooses a single random key from a map of keys and weights.
+// nolint:unused
```

Comment:
Can we get rid of these if they're not used or do you see them being used in the future?

Reply:
I can't see any use for them in the near future but I was hesitant about removing it - or removing anything for that matter :) - so I just added the nolint

Reply:
Don't have a strong preference here, but the less code to maintain, the better :)